mne/viz/evoked.py

```diff
@@ -1633,7 +1684,11 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
         ymin, ymax = ylim.get(ch_type, [None, None])
 
     # deal with dict/list of lists and the CI
-    if ci is not None and not (isinstance(ci, np.float) or callable(ci)):
+    if ci is None:
+        ci = False
+    if ci is True:
+        ci = .95
+    elif ci is not False and not (isinstance(ci, np.float) or callable(ci)):
         raise TypeError('ci must be float or callable, got ' + str(type(ci)))
```

Comment:
nitpick: "ci must be None, bool, float or callable ..."

Reply:
Addressed, merge if you want.